bin/brew

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 
-chdir() {
-  cd "$@" >/dev/null || {
-    echo "Error: failed to cd to " "$@" "!" >&2
+odie() {
+    if [[ -t 2 ]]
+    then
+        echo -ne "\033[4;31mError\033[0m: " >&2
```

Comment:
> but I'd like to see the colon after “Error” also colored.

We didn't do this for ruby version of `odie`. So I will keep it for the time being.